### PR TITLE
Add recenter command and an option for auto recenter

### LINKED ISSUE #12: Keep current match in the middle of the window?
Sometimes we are just searching to peek things. When the match is on the beginning/end of the window, we have to search backward/forward again and then move back to show its surroundings. So how about keeping current match in the middle of the window?

### PR DESCRIPTION
**Pull request by AmaiKinono**
Closes #12.

### CODE REVIEW

File: ctrlf.el [hunk 1]
```diff
@@ -387,7 +399,9 @@ fails, return nil, but still move point."
   ;; Next search should go forward.
   (setq ctrlf--backward-p nil)
   ;; Force recalculation of search.
-  (setq ctrlf--last-input nil))
+  (setq ctrlf--last-input nil)
+  (when ctrlf-auto-recenter
+    (ctrlf-recenter)))
```
Comment (raxod502) [inline ctrlf.el]: To avoid duplication, can we do this in the `ctrlf--minibuffer-post-command-hook` instead?

**Review by raxod502 — CHANGES_REQUESTED** [review]
Looks good except for one thing.

**Comment by raxod502** [comment]
Oh also there is CI for this project which accidentally wasn't enabled, like for Selectrum. I've fixed that now.

File: ctrlf.el (hunk 1 above)
Comment (AmaiKinono) [reply ctrlf.el]: Done.

**Review by raxod502 — APPROVED** [review]
Looks good to me. Thank you!

**Pull request merged**